doc/Programming-guide/Key-platform-features/Data-collection/Couchbase-log-appender/index.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: page
+title: Couchbase log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Couchbase-log-appender
+sort_idx: 30
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Couchbase log appender is responsible for transferring logs from the Operations service to the Couchbase storage. Logs are stored in document storage.
+
+Each log document consists of the following fields:  
+
+where:
+
+* <i>id</i> - randomly generated UUID string
+* <i>header</i> - includes log's metainformation: <i>endpointKeyHash</i> (a key hash identifying the endpoint which produced the log record), 
+<i>[applicationToken]({{root_url}}Glossary)</i> (matches the token of the current application), <i>headerVersion</i> (currently is 1), <i>timestamp</i> (a timestamp in milliseconds
+when logs were uploaded to the key/value storage)
+* <i>event</i> - log body (log item, received from a client)
+
+Values are stored as serialized Generic Records using record wrapper avro schema.
+
+# Creating Couchbase log appender in Admin UI
+
+The easiest way to create a Couchbase log appender for your application is by using Admin UI.
+
+To create a log appender of the Couchbase storage type, do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender</b>.
+2. Enter the log appender name and description, select the minimum and maximum supported schema version, select necessary log metadata fields.
+3. Set the log appender type to <i>Couchbase</i>.
+4. Fill in the Couchbase server list and other fields.
+5. Click <b>Add</b>.
```

Comment:
Please identify what the log appender status after the clicking **Add**. Few words like 'Log appender is ready and operational at this point' will be enough.